BD-2057 pt2: Clarify only HTML IAMs don't need special libraries (#6118)
braze-inc/braze-docs: _includes/archive/android/gifs.md

{% if include.channel=="in-app messages" %}

> Braze supports displaying animated GIFs and SVG images by default for HTML in-app messages. A custom image library is not required, but it is supported.

Reviewer: ```suggestion
Braze supports displaying animated GIFs and SVG images by default for HTML in-app messages. A custom image library is not required, but it is supported.

For non custom HTML in-app messages, you must follow the guide below.
```

Reviewer 2: Instead of "For non custom HMTL", can we just say "For non-HTML in-app messages"?  It removes confusion if people are using a template, etc.

> Braze supports displaying animated GIFs and SVG images by default for HTML in-app messages. A custom image library is not required, but it is supported.

Reviewer 2: ```suggestion
Braze supports displaying animated GIFs and SVG images by default for HTML in-app messages. A custom image library is required when using non-HTML in-app messages.
```
